lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,24 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
-      origin_branch = current_branch
-      run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
-      run_command_with_label "git checkout #{origin_branch}"
+    def update_destination(options = {})
+      feature_branch = options[:feature_branch]
+      base_branch = options[:base_branch]
+      remote = options[:remote] || 'origin'
+      run_command_with_label "git checkout #{base_branch}"
+      run_command_with_label "git pull #{remote} #{base_branch}"
+      run_command_with_label "git checkout #{feature_branch}"
+    end
+
+    def update_feature_branch(options = {})
```

Comment:
It seems like we're doing too much here.  I don't think it makes sense to update the base branch (since we do that during delivery).  All we really need to do is make sure the feature branch is synced with remote, and then pull in any upstream changes.  That can be done in two commands: `git pull origin feature-branch && git pull remote base-branch`.  Objections?